freeimage pkg-config should include -lstdc++

I'm trying to add a [rucksack](https://github.com/andrewrk/rucksack) package and I'm getting this:

```
/home/andy/dev/mxe/usr/bin/i686-w64-mingw32.static-gcc  -O3 -DNDEBUG    -Wl,--whole-archive CMakeFiles/rucksack.dir/objects.a -Wl,--no-whole-archive  -o rucksack.exe -Wl,--out-implib,librucksack.dll.a -Wl,--major-image-version,0,--minor-image-version,0  librucksack.a -llaxjson -lfreeimage -lws2_32 -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x72c): undefined reference to `_imp__FreeImage_Initialise@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x741): undefined reference to `_imp__FreeImage_DeInitialise@0'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x179d): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x17fb): undefined reference to `_imp__FreeImage_GetFileType@8'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x1825): undefined reference to `_imp__FreeImage_FIFSupportsReading@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x1844): undefined reference to `_imp__FreeImage_Load@12'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x185b): undefined reference to `_imp__FreeImage_HasPixels@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x1897): undefined reference to `_imp__FreeImage_GetWidth@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x18a6): undefined reference to `_imp__FreeImage_GetHeight@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x2775): undefined reference to `_imp__FreeImage_Allocate@24'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x2788): undefined reference to `_imp__FreeImage_GetBits@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x2798): undefined reference to `_imp__FreeImage_GetPitch@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x27d9): undefined reference to `_imp__FreeImage_GetBPP@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x27ed): undefined reference to `_imp__FreeImage_ConvertTo32Bits@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x27fe): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x28a6): undefined reference to `_imp__FreeImage_OpenMemory@8'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x28d1): undefined reference to `_imp__FreeImage_SaveToMemory@16'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x28dd): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x28fc): undefined reference to `_imp__FreeImage_AcquireMemory@12'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x2e71): undefined reference to `_imp__FreeImage_CloseMemory@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x36cd): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x3762): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x37ae): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x3bc5): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x3c39): undefined reference to `_imp__FreeImage_Unload@4'
librucksack.a(rucksack.c.obj):rucksack.c:(.text+0x3c8d): more undefined references to `_imp__FreeImage_Unload@4' follow
```

Full log: http://paste.ubuntu.com/10155662/

Here is my package so far: https://github.com/andrewrk/mxe/compare/help-rucksack

Any ideas?

Possibly related: #158 

The only other package that depends on freeimage is cegui and that uses autotools instead of cmake so it's tricky to use as an example.

--- a/src/freeimage.mk
+++ b/src/freeimage.mk
@@ -36,7 +36,7 @@ define $(PKG)_BUILD
      echo 'Version: $(freeimage_VERSION)'; \
      echo 'Description: FreeImage'; \
      echo 'Cflags: -DFREEIMAGE_LIB'; \
-     echo 'Libs: -lfreeimage -lws2_32';) \
+     echo 'Libs: -lfreeimage -lws2_32 -lstdc++';) \
      > '$(PREFIX)/$(TARGET)/lib/pkgconfig/freeimage.pc'
 
     '$(TARGET)-gcc' \

